tests/ssl-opt.sh

```diff
@@ -2332,7 +2341,7 @@ run_test    "Opaque key for server authentication: invalid alg: ecdh with RSA ke
 requires_config_enabled MBEDTLS_USE_PSA_CRYPTO
 requires_config_enabled MBEDTLS_X509_CRT_PARSE_C
 requires_hash_alg SHA_256
-requires_config_enabled MBEDTLS_CCM_C
+requires_config_enabled PSA_WANT_ALG_CCM
```

Comment:
Is this line required? I thought that the automatic requirements for the cipher suite would take care of it.